src/main/java/it/gov/pagopa/payhub/activities/connector/processexecutions/config/IngestionFlowFileApisHolder.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.payhub.activities.connector.processexecutions.config;
+
+import it.gov.pagopa.pu.processexecutions.client.generated.IngestionFlowFileEntityControllerApi;
+import it.gov.pagopa.pu.processexecutions.client.generated.IngestionFlowFileEntityExtendedControllerApi;
+import it.gov.pagopa.pu.processexecutions.generated.ApiClient;
+import it.gov.pagopa.pu.processexecutions.generated.BaseApi;
+import jakarta.annotation.PreDestroy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Lazy
+@Service
+public class IngestionFlowFileApisHolder {
+
+    private final IngestionFlowFileEntityControllerApi ingestionFlowFileEntityControllerApi;
+    private final IngestionFlowFileEntityExtendedControllerApi ingestionFlowFileEntityExtendedControllerApi;
+
+    private final ThreadLocal<String> bearerTokenHolder = new ThreadLocal<>();
+
+    public IngestionFlowFileApisHolder(
+            @Value("${rest.classification.base-url}") String baseUrl,
```

Comment:
this is not right, update also the application.yml and use the env variable PROCESS_EXECUTIONS_BASE_URL
```suggestion
            @Value("${rest.process-executions.base-url}") String baseUrl,
```